pupil_src/shared_modules/raw_data_exporter.py

```diff
@@ -102,162 +102,254 @@ class Raw_Data_Exporter(Analysis_Plugin_Base):
         gaze_normal1_x - x normal of the visual axis for eye 1 in the world camera coordinate system (not avaible for monocular setups.). The visual axis goes through the eye ball center and the object thats looked at.
         gaze_normal1_y - y normal of the visual axis for eye 1
         gaze_normal1_z - z normal of the visual axis for eye 1
-        '''
+        """
+
     icon_chr = chr(0xe873)
-    icon_font = 'pupil_icons'
+    icon_font = "pupil_icons"
+
+    def __init__(
+        self,
+        g_pool,
+        should_export_pupil_positions=True,
+        should_export_field_info=True,
```

Comment:
I know this is terribly late but I wonder. The initial motivation for adding this complexity was that exports a large/take a long time. This reason does not apply to writing the filed info. Why is this a setting then. Whats the cost of always doing this?

Reply:
@papr

Reply:
I know this is nit-picky but I just want to drive home the point that every slider and setting comes at a cost and need justification.